core/domain/feedback_services_test.py

```diff
@@ -232,7 +232,52 @@ def test_get_all_threads_without_suggestion(self):
             with self.swap(utils, 'get_time_in_millisecs',
                            self._get_milliseconds):
                 threads = feedback_services.get_all_threads(
-                    self.EXP_ID2, False)
+                    self.EXP_ID1, False)
```

Comment:
I didn't mean that using the self.swap is wrong, it does its job, so it should be fine.
What I meant by logically wrong test is:

fake get_threads(i.e self._get_threads) returns: 
`
['exp1.thread1', 'exp1.thread2', 'exp1.thread3', 'exp2.thread4', 'exp2.thread4']. 
`
After that we try to call:
`
feedback_services.get_all_threads('exp2')
`
Which in a consequence calls get_threads('exp2') which returns fake get_threads.
In a result, we will receive all threads including 'exp1.thread1' and other. 
I think this is logically wrong.
That is why I've changed _get_threads to return only 'exp1.*' and tests to use get_all_threads(exp1.*).

I was able to see the authors intends, but in my opinion current version more correct. 

In the end, I would prefer `for thread in self.thread thread.put()` though.
In later case authors intends would be correct.